docs/crio.conf.d.5.md

```diff
@@ -0,0 +1,25 @@
+% crio.conf.d(5)
```

Comment:
Can we add this man page to the bundle as well:
https://github.com/cri-o/cri-o/blob/5996f36f459ea2143b50dc98468e27ef69487e0f/bundle/build#L17-L20

https://github.com/cri-o/cri-o/blob/5996f36f459ea2143b50dc98468e27ef69487e0f/bundle/Makefile#L34-L35

I can also follow-up on that.

Reply:
+1

Reply:
Did that now. Thanks for the hint.